modules/default/alert/alert.js

```diff
@@ -79,7 +79,7 @@ Module.register("alert", {
 
 		//If module already has an open alert close it
 		if (this.alerts[sender.name]) {
-			this.hide_alert(sender);
+			this.hide_alert(sender, false);
```

Comment:
that would now behave differently to what the comment line above says? if so, wouldnt that mean we have change in behaviour for the alert module (which would surpise other users)?

Reply:
It's not so different, actually. Still closing the previous notification, the only change is that doesn't send the close action twice.
onClose function does the same as hiding it. And it's called as a delayed callback when the next notification take place, that's hiding not only the previous one, but the new one as well.

I would like to iterate above the alert module code when I get some time, because there are some functions that doesn't do much. Right now, this is more like a hotfix than an actual fix to the problem.

Reply:
Good to hear that you want to clean it up :-)

Reply:
Maybe add some explanation to the code would be nice for this PR too. But I am not opposed on merging it if it helps fix your bugs and doesnt break any exisiting behaviour